docs/docs/projects/README.md

```diff
@@ -77,8 +79,72 @@ At this time it's not possible to move workflows out of GitHub Synchronized Proj
 
 :::
 
+## Access permissions
+
+### Understanding who has access to which project
+We've introduced 2 new columns in the [project list view](https://pipedream.com/projects): **Owner** and **Access**.
+
+**Owner** indicates who within the workspace owns each project. This is typically the person who created the project.
+
+![Project Listing (Owner)](./images/project-listing-owner.png)
+
+::: warning Some projects may not have owners
+Projects created before February 2024 don't automatically have owners.
+:::
+
+**Access** indicates who within the workspace has access to each project, and this can be displayed as "me", "Workspace", or "N members".
+
+![Project Listing (Access)](./images/project-listing-access.png)
+
+### Permissions
+For workspace members who have access to the project (either because it's shared with the entire workspace or with that member specifically), below are the permissions for the project.
+
+| Operations | Project creator | Workspace members |
+| --  | :--: | :--: | :--: |
+| View in [projects listing](https://pipedream.com/projects) | :white_check_mark: | :white_check_mark: |
+| View in [Event History](https://pipedream.com/event-history) | :white_check_mark: | :white_check_mark: |
+| View in global search | :white_check_mark: | :white_check_mark: |
+| Manage project workflows | :white_check_mark: | :white_check_mark: |
+| Manage project files | :white_check_mark: | :white_check_mark: |
+| Manage project variables | :white_check_mark: | :white_check_mark: |
+| Manage project access | :white_check_mark: | :x: |
```

Comment:
```suggestion
| Manage member project access | :white_check_mark: | :x: |
```

Let me know if that's not accurate, but otherwise being more specific.

Reply:
Yea good call. Do you think `Manage member access` is sufficient? Is it clear that we're in the context of a project? I guess same question for all of the rows where I mention **projects** explicitly. Would be nice to not be so redundant if possible.